templates/deployments.yaml

```diff
@@ -1824,6 +1829,11 @@ spec:
           configMap:
             name: {{ .Release.Name }}-st2-rbac-mappings
         {{- end }}
+        {{- if .Values.st2.overrides }}
+        - name: st2-overrides-vol
+          configMap:
+            name: {{ .Release.Name }}-st2-override-configs
+        {{- end }}
```

Comment:
Can you use your helper template instead here?

Reply:
Used the template here.